app/src/components/navigation/TopBar.vue

```diff
@@ -8,7 +8,7 @@ const route = useRoute();
 
 <template>
     <header>
-        <div class="z-40 bg-white drop-shadow dark:bg-zinc-700">
+        <div class="z-40 bg-white drop-shadow dark:bg-slate-800">
```

Comment:
I feel the top bar is a tiny bit too light still and that makes the contrast between the background and top bar a bit much, but that is a limitation in _**tailwindCSS**_ as there is no `bg-slate-850`. This could also just be my personal preference.